lektor/admin/static/webpack.config.js

```diff
@@ -22,41 +23,54 @@ module.exports = {
     filename: '[name].js'
   },
   devtool: 'source-map',
+  optimization: {
+    splitChunks: {
+      chunks: 'all',
+      name: 'vendor'
```

Comment:
Is there a reason why we need two CSS files now instead of one like before?

Reply:
Well I tried to bundle the CSS into one with the commented part. I assume it's still possible with Webpack 4, but I don't see the point. If we split the JS into two, it makes just as much sense to also split the CSS.

Reply:
Fair enough, I don't mind this.